### ISSUE TITLE
Vanilla Functionality

### ISSUE THREAD

**Issue by ItsMoorbinTime** [AndreyMrovol/LethalTerminalFormatter#3]
TL;DR Many thanks for making this mod, I hope vanilla compatibility will be introduced in a future update/new branch!

One of the dependencies for "LethalTerminalFormatter" is "LethalLevelLoader" https://thunderstore.io/c/lethal-company/p/IAmBatby/LethalLevelLoader/ .

I am almost certain this is so that "LethalTerminalFormatter" can work when other modded moons are installed, which is a great idea. However, since "LethalLevelLoader" is a server-side mod, it creates problems when trying to play a vanilla/lightly modded experience:

- When trying to join/host a game with friends, everyone must have "LethalLevelLoader" installed, not much of a problem
- When trying to join/host a game for randoms, everyone must have "LethalLevelLoader" installed. The only people who install "LethalLevelLoader" will naturally have other mods installed to go to modded moons/expect to join a game where they can go to modded moons.

Uninstalling "LethalLevelLoader" introduces bugs (attached screenshot), thus I ask you: Can a version/branch of this mod be created such that one can use it without server-side mods, thereby allowing one to play a vanilla experience with this newly formatted terminal?

====================================================================================
IMAGES WHEN "LethalLevelLoader" IS NOT INSTALLED

Data about moons from the terminal instead displays variable names.
![20240224212443_1](https://github.com/AndreyMrovol/LethalTerminalFormatter/assets/126391665/7195b898-0be0-41ec-b97b-17d5f3c5b8c8)

------------------------------------------------------------------------------------------------------------------------------------------------------

Store data appears to be unaffected.
![20240224212451_1](https://github.com/AndreyMrovol/LethalTerminalFormatter/assets/126391665/57b4541a-4390-4129-8ff5-61e3d16e974d)

====================================================================================

**Comment by AndreyMrovol** [comment]
Hey, thanks for your feedback! I'm planning to make the mod function without LLL, but there's no ETA (as of now).
This will be added in next release 😅

**Comment by AndreyMrovol** [comment]
Hey, i've just pushed an update to Thunderstore which allows you to use modified moon list without LLL.

Note: its values are hardcoded (so it won't update with AC and other mods changing the cost), and will likely break with future game update, but it was a relatively easy feature to add, so why not :D

Please check it out and let me know about any issues!

**Comment by ItsMoorbinTime** [comment]
The update works brilliantly... For vanilla. I think the update breaks the moons list when using modded moons and "LethalLevelLoader". None of the modded moons display costs (and possibly weathers).

I believe it would be a good idea to release a fork of the mod for vanilla, and another for modded moons and "LethalLevelLoader".Then again, I'm not experienced with creating mods for Lethal Company, so perhaps its possible to have 1 mod that works in vanilla AND with modded moons.

**Comment by AndreyMrovol** [comment]
> I think the update breaks the moons list when using modded moons and "LethalLevelLoader". None of the modded moons display costs (and possibly weathers).

Are you using AdvancedCompany?

**Comment by ItsMoorbinTime** [comment]
> Are you using AdvancedCompany?

I just came back to mention this.
It'll be because of that whole drama with the AC dev implementing LLL into his own mod, the implementation has been done in such a way that this mod and AC incompatible.

**Comment by AndreyMrovol** [comment]
> > Are you using AdvancedCompany?
> 
> I just came back to mention this. It'll be because of that whole drama with the AC dev implementing LLL into his own mod, the implementation has been done in such a way that this mod and AC incompatible.

Yeah, sadly latest AC versions are incompatible with LLL (and by that my mod) - you can tweak prices manually (using https://github.com/AndreyMrovol/LethalTerminalFormatter/issues/6#issuecomment-1967610416), this is something outside my mod's control.
